src/AST-Core/RBVariableNode.class.st

```diff
@@ -117,18 +117,18 @@ RBVariableNode >> isArgumentVariable [
 	^variable isArgumentVariable
 ]
 
-{ #category : #testing }
-RBVariableNode >> isBlockVar [
-	"true if a variable node is defined by a block node"
-	^variable scope isBlockScope
-]
-
 { #category : #testing }
 RBVariableNode >> isClassVariable [
 	^variable isClassVariable
 
 ]
 
+{ #category : #testing }
+RBVariableNode >> isDefinedByBlock [
+	"true if a variable node is defined by a block"
```

Comment:
Some comments look really useless when it's just a split of words in the class name and methods by space

Reply:
Actually it starts to be useless with rename